tree/ntuple/v7/test/ntuple_test.hxx

```diff
@@ -114,6 +114,8 @@ using RPrepareVisitor = ROOT::Experimental::RPrepareVisitor;
 using RPrintSchemaVisitor = ROOT::Experimental::RPrintSchemaVisitor;
 using RRawFile = ROOT::Internal::RRawFile;
 using EContainerFormat = RNTupleFileWriter::EContainerFormat;
+template <typename T>
+using RNTupleView = ROOT::Experimental::RNTupleView<T>;
```

Comment:
As Florine remarked in https://github.com/root-project/root/pull/17598#discussion_r1939166623, "This change seems unrelated to the rest of commit."